lib/Config.php

```diff
@@ -153,6 +159,23 @@ public function isRecordingEnabled(): bool {
 		return $isSignalingOk && $recordingEnabled;
 	}
 
+	public function getRecordingFolder(string $userId): string {
+		return $this->config->getUserValue($userId, 'spreed', 'recording_folder', '/Recording');
+	}
+
+	public function getRecordingAllowedMimes(): array {
```

Comment:
This will not be necessary. We know exactly what the tool will create, or we shouldn't check it.

Reply:
I implemented this to make the API more flexible and allow to developers of other apps or services to implement our own recording systems and upload the artifacts to recording API.

Today already exists an app that record the call. If the developer of this app want to send our recording files to API and trigger with this all the features related of this endpoint, will be possible do this changing this setting and adding the extensions and mime that the recording process generate if is different of the default  accepted by Talk.

Reply:
Maybe @danxuliu could help us with opinion from the integration side.

Reply:
Well, remember that I gave you the same arguments as nickvergessen, but then I agreed about making it configurable due to the _This can be used to accept different extensions and mime types if a different recording tool is used_, so... there is not much I can add :-P

If compatibility with possible additional recorders is desired then it should be configurable (or just trust the recorders and accept whatever extension/mime is given without checking it). Otherwise the extension/mime could be just hardcoded and checked to ensure that the default recorder (or whoever has the credentials to upload to that endpoint) is behaving as expected, or again just trust it without checking.